br/pkg/lightning/importer/import.go

```diff
@@ -116,6 +116,8 @@ const (
 var (
 	minTiKVVersionForDuplicateResolution = *semver.New("5.2.0")
 	maxTiKVVersionForDuplicateResolution = version.NextMajorVersion()
+	minTiDBCurrentResourceGroupVersion   = *semver.New("6.6.0")
+	maxTiDBCurrentResourceGroupVersion   = version.NextMajorVersion()
```

Comment:
should we skip v7.0.0/v7.1.0 and do they support `select current_resource_group();`

Reply:
After checking, the function is supported after v7.0.0, so we can set min to 7.7.0.(will modify)
https://github.com/pingcap/tidb/pull/41722

Reply:
What about just ignore the error is `select current_resource_group();` returns an error like:
<img width="579" alt="image" src="https://github.com/pingcap/tidb/assets/5196885/7943a67c-973e-44e9-a31b-27886033e089">